python/adbc_driver_snowflake/adbc_driver_snowflake/__init__.py

```diff
@@ -90,8 +90,11 @@ class DatabaseOptions(enum.Enum):
 class StatementOptions(enum.Enum):
     """Statement options specific to the Snowflake driver."""
 
-    #: The number of rows per batch. Defaults to 1024.
-    BATCH_ROWS = "adbc.rpc.result_queue_size"
```

Comment:
Did you mean to remove this?

Reply:
yea, because that name and description are completely incorrect with respect to that option....

Reply:
It broke a test though:

```

  /home/runner/work/arrow-adbc/arrow-adbc/python/adbc_driver_snowflake/tests/test_lowlevel.py:52:56 - error: Cannot access member "BATCH_ROWS" for type "type[StatementOptions]"
    Member "BATCH_ROWS" is unknown (reportGeneralTypeIssues)
```